package.json

```diff
@@ -45,7 +45,7 @@
     "@types/node": "^18.15.2",
     "@types/pg": "^8.6.6",
     "eslint": "^8.36.0",
-    "eslint-config-metarhia": "^8.1.0",
+    "eslint-config-metarhia": "^8.2.0",
     "eslint-config-prettier": "^8.7.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-prettier": "^4.2.1",
```

Comment:
```suggestion
    "eslint-plugin-prettier": "^5.0.0-alpha.2",
```

Reply:
- Also update prettier to `^3.0.0`
- Not necessarily, but taking the opportunity update eslint to `^8.44.0`

Reply:
Use `npm audit` and even `npm audit fix --force` if needed before commit.

Reply:
Going to land after this last dependencies fixes